### PR TITLE
Entitylabel bug fix

### PR DESCRIPTION
**Pull request by 84634E1A607A**
1. Found that multicellular microbes may not have Species, causing a crash.
2. Commented out NaN PrintErr because it costs too much time. (15FPS before -> 60FPS after)

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: src/engine/DebugOverlays.EntityLabel.cs [hunk 1]
```diff
@@ -47,7 +47,8 @@ private void UpdateEntityLabels()
                 {
                     if (microbe.Species != null)
                     {
-                        label.Text = $"[{microbe.Name}:{microbe.Species.Genus[0]}.{microbe.Species.Epithet.Left(4)}]";
+                        label.Text =
+                            $"[{microbe.Name}:{microbe.Species.Genus.Left(1)}.{microbe.Species.Epithet.Left(4)}]";
```
Comment (Kasterisk) [inline src/engine/DebugOverlays.EntityLabel.cs:51]: Why does adding `Left` fixes the crash?

Comment (84634E1A607A) [reply src/engine/DebugOverlays.EntityLabel.cs:51]: Directly use [0] triggers an exception, but Left(1) returns string.empty if there isn't any char.

Comment (84634E1A607A) [reply src/engine/DebugOverlays.EntityLabel.cs:51]: When I coded this part, I assumed that the species may be less than four characters, so I used left(4), But I assume that there will be at least one character in genus, so I directly used [0].

Comment (Kasterisk) [reply src/engine/DebugOverlays.EntityLabel.cs:51]: Ah okay.

Comment (hhyyrylainen) [reply src/engine/DebugOverlays.EntityLabel.cs:51]: How does there exist species with no genus? Isn't that *very* invalid data?

Comment (84634E1A607A) [reply src/engine/DebugOverlays.EntityLabel.cs:51]: In multicellular editor, the cell on display.

Comment (hhyyrylainen) [reply src/engine/DebugOverlays.EntityLabel.cs:51]: Ah okay, I suppose that is going to be different. I guess I'll accept this then here as this is a debugging feature.

**Review by hhyyrylainen — APPROVED** [review]
Seems fine to me. I commented on the nan compounds issue that this PR comments out the error print and anyone wanting to debug the issue should uncomment the code.

**Pull request merged**